src/pages/index.js

```diff
@@ -1,89 +1,241 @@
+  validationSettings,
+  variables.addNewModalCard);
+addFormValidator.enableValidation();
 
-const profileEditForm = profileEditModal.querySelector(".modal__form");
 const editFormValidator = new FormValidator(
-  profileEditForm,
-  validationSettings
-);
+  validationSettings, 
+  variables.profileEditModal);
+editFormValidator.enableValidation();
 
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+const avatarFormValidator = new FormValidator(
+  validationSettings, 
+  variables.avatarModal);
+avatarFormValidator.enableValidation();
 
-profileEditButton.addEventListener("click", () => {
-  const profileUserInfo = userInfo.getUserInfo();
-  profileTitleInput.value = profileUserInfo.title;
-  profileDescriptionInput.value = profileUserInfo.description;
-  profileEditFormPopup.open();
-});
+// ADD FORM
+
+const addPopup = new PopupWithForm(
+  variables.addNewModalCard, 
+  handleAddCardSubmit);
+
+function openAddForm() {
+  addPopup.open();
+}
+
+addPopup.setEventListeners();
 
-const profileEditFormPopup = new PopupWithForm(
-  "#profile-edit-modal",
-  handleProfileEditFormSubmit
+variables.profileAddButton.addEventListener(
+  "click", () => openAddForm());
+
+// EDIT FORM
+
+const editPopup = new PopupWithForm(
+  variables.profileEditModal,
+  handleEditFormSubmit
 );
-profileEditFormPopup.setEventListeners();
 
-document.querySelector(".profile__plus-button").addEventListener("click", () => {
-  cardFormValidator.toggleButtonState();
-  addCardPopup.open();
+function openEditForm() {
+  const user = userInfo.getUserInfo();
+  variables.profileTitleInput.value = user.name;
+  variables.profileDescriptionInput.value = user.description;
+  editPopup.open();
+}
+
+editPopup.setEventListeners();
+
+variables.profileEditButton.addEventListener("click", () => {
+  openEditForm();
 });
 
-function handleAddCardFormSubmit(formData) {
-  const card = createCard({ name: formData.title, link: formData.url });
-  cardSection.addItem(card);
-  addCardPopup.close();
-}
+// AVATAR FORM
 
-const addCardPopup = new PopupWithForm(
-  "#add-card-modal",
-  handleAddCardFormSubmit
+const avatarModal = new PopupWithForm(
+  variables.avatarModal,
+  handleAvatarFormSubmit
 );
-addCardPopup.setEventListeners();
+avatarModal.setEventListeners();
 
-function createCard(cardData) {
-  return new Card(cardData, "#card-template", () => {
-    popupWithImage.open(cardData.link, cardData.name);
-  }).getView();
+function openAvatarForm() {
+  avatarModal.open();
 }
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardListEl = createCard(cardData);
-      cardSection.addItem(cardListEl);
-    },
-  },
-  ".cards__list"
-);
+variables.profileImage.addEventListener("click", () => {
+  openAvatarForm();
+});
 
-cardSection.renderItems();
+function handleAvatarFormSubmit(inputValues) {
+  setButtonText(variables.avatarModalButton, "Saving...");
```

Comment:
Needs correction: the text on the submit button disappears when I submit the form